src/ledger/LedgerManagerImpl.h

```diff
@@ -38,7 +38,7 @@ class Database;
 class LedgerTxnHeader;
 class BasicWork;
 
-class SorobanLedgerMetrics
+class SorobanMetrics
```

Comment:
Please move this class to a separate file (similar to `OverlayMetrics`, for example): this class has gotten quite bit, and LedgerManager is already a very large module

Reply:
Done.